core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -217,7 +217,7 @@ private[spark] class ExecutorAllocationManager(
    * the scheduling task.
    */
   def start(): Unit = {
-    listenerBus.addListener(listener)
+    listenerBus.addToQueue(listener, LiveListenerBus.EXECUTOR_MGMT_QUEUE)
```

Comment:
will we have more queues in the future? If it's not too many, maybe we can use different methods, like `addToMGMTQueue`